packages/cosmos_ui_components/lib/components/cosmos_image_button.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:cosmos_ui_components/cosmos_ui_components.dart';
+import 'package:flutter/material.dart';
+
+class CosmosImageButton extends StatelessWidget {
+  final String assetUrl;
```

Comment:
its rather a path than url, but you can just simply name it `asset`, like `Image.asset()` does

Reply:
Updated